app/views/play/modal/PollModal.coffee

```diff
@@ -86,20 +86,28 @@ module.exports = class PollModal extends ModalView
       
       # The following block allows for the user to be indecisive with their answer, updating UI accordingly.
       btn = @$el.find('.btn.btn-illustrated.btn-lg.done-button')
+      btn.off('click')
+
       if nextPollId
         btn.text(i18n.t('common.next'))
-        btn.one('click', ()=>
+        btn.one('click', () =>
           btn.prop('disabled', true);
-
-          # Show live Class Modal
           if @poll.id is "5e84c7a519cb270028516687"
+            # Show live Class Modal
             @trigger('trigger-show-live-classes')
           else
             @trigger('trigger-next-poll', nextPollId)
         )
       else
         btn.text(i18n.t('play_level.done'))
-        btn.off('click')
+
+        # For code quest modal
+        btn.one('click', () =>
+          # TODO: Replace with production poll id once it is ready.
+          if @poll.id is "5f2058074cbd8c19861848fc"
```

Comment:
TODO: Add this poll with `hidden: true` to CodeCombat and then add the id generated here.

Once this code is merged, set hidden to false.